lib/grammers-client/src/types/input_message.rs

```diff
@@ -134,6 +134,23 @@ impl InputMessage {
         self
     }
 
+    /// Include an external photo in the message.
+    ///
+    /// The server will download and compress the image and convert it to JPEG format if
+    /// necessary.
```

Comment:
```suggestion
    /// The Telegram server will download and compress the image and convert it to JPEG format if necessary.
```

Reply:
I copied the text from the `.photo()` method. Should I add the "Telegram" there too?

Reply:
Oh, didn't even realize. Sure.